src/main/java/com/esri/samples/scene/add_a_point_scene_layer/README.md

```diff
@@ -0,0 +1,41 @@
+<h1>Add a point scene layer</h1>
+
+<p>View a point scene layer from a scene service.</p>
+
+<p><img src="AddAPointSceneLayer.png"/></p>
+
+<h2>Use case</h2>
+
+<p>Point scene layers can efficiently display large amounts of point features. While point cloud layers can only display simple symbols, point scene layers can display any type of billboard symbol or even 3D models, as long as the location of the symbol can be described by a point. Points are cached and automatically thinned when zoomed out to improve performance.</p>
+
+<h2>How to use the sample</h2>
+
+<p>Pan around the scene and zoom in. Notice how many thousands of additional features appear at each successive zoom scale.</p>
```

Comment:
it also shows features 'behind' the active face of the globe (e.g. looking on Africa airports from Australia are visible behind): is this intentional?

Reply:
That definitely sounds like a bug.

Reply:
I reproduced it with 2328 (super trippy!) and then saw it looks fixed with the latest build 2358. Could you confirm it looks okay to you with that build?

Reply:
Oh whoops, I must have been checking it with the out of date build. With 2358 it looks good now!